### ISSUE TITLE
Re. Token Designer Usage in VSCode Related

### ISSUE THREAD

**Issue by pvrbharg** [InterWorkAlliance/visual-token-designer#5]
Hi,

I am trying to model the token design and use it in the tool. However - I am unclear as to how I open a sample project in a sandbox environment and understand the model - as shown in the documentation screen shot for carbon emissions.

Please see the two attachments and let me know what I am missing and what I need to do to get the point shown in the screen shot.

![EmptyTokenDesignScreen-Windows-Linux-Platforms](https://user-images.githubusercontent.com/5838902/121782791-4855dc80-cb79-11eb-8fff-c895c2c5be3a.png)
![CarbonEmissionTokenDesignScreen](https://user-images.githubusercontent.com/5838902/121782792-48ee7300-cb79-11eb-9bea-c56d824a5827.png)

**Comment by pvrbharg** [comment]
Hi,

I realized I needed to be on TTK v1.0.2 release and Token Designer v1.0.7 release.

Still things are not working for me as indicated in screen captures. I am also finding instructions to be incorrect at some points in documentation.

Should I be trying this code base or wait for some time?

![2021-06-12_17-23-36](https://user-images.githubusercontent.com/5838902/121789608-4f92df80-cba5-11eb-83d5-e7c10f3e39cc.png)
![2021-06-12_17-25-31](https://user-images.githubusercontent.com/5838902/121789609-502b7600-cba5-11eb-8d4e-8ab888b5ddf5.png)
![2021-06-12_17-36-09](https://user-images.githubusercontent.com/5838902/121789610-502b7600-cba5-11eb-893d-d6c81919cb74.png)
